<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIOutput</class>
 <widget class="pDockWidget" name="UIOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/MessageBox.qrc">
    <normaloff>:/icons/taboutput.png</normaloff>:/icons/taboutput.png</iconset>
  </property>
  <property name="windowTitle">
   <string>Output</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lRawCommand">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Send Raw Command :</string>
      </property>
      <property name="indent">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cbRawCommand">
      <property name="toolTip">
       <string>You can enter here a command and press return to execute it.</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="tbOutput">
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="tabStopWidth">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pDockWidget</class>
   <extends>QDockWidget</extends>
   <header location="global">widgets/pDockWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/MessageBox.qrc"/>
 </resources>
 <connections/>
</ui>
